src/main/java/study/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,45 @@
+package study.lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public enum LottoRank {
+    FIRST(2000000000, 6),
```

Comment:
```suggestion
    FIRST(2_000_000_000, 6),
```
와 같이 사용해서 가독성을 높혀줄 수 있어요 : )